src/deep-kernel/lib/Config/Loader.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Created by AlexanderC on 3/7/16.
+ */
+
+'use strict';
+
+import {KernelDriver} from './Driver/KernelDriver';
+import {AsyncConfig} from './Driver/AsyncConfig';
+
+export class Loader {
+  /**
+   * @param {AbstractDriver|FsDriver|HttpDriver|KernelDriver|ComplexDriver|*} driver
+   */
+  constructor(driver = null) {
+    this._driver = driver;
+  }
+
+  /**
+   * @param {Kernel|*} kernel
+   */
+  static asyncConfigLoader(kernel) {
+    return new Loader(AsyncConfig.createFromKernel(kernel));
+  }
+
+  /**
+   * @param {Kernel|*} kernel
+   */
+  static kernelLoader(kernel) {
+    return new Loader(new KernelDriver(kernel));
+  }
+
+  /**
+   * @returns {AbstractDriver|FsDriver|HttpDriver|KernelDriver|ComplexDriver|*}
+   */
+  get driver() {
+    return this._driver;
+  }
+
+  /**
+   * @param {AbstractDriver|FsDriver|HttpDriver|KernelDriver|ComplexDriver|*} driver
+   * @returns {Loader}
+   */
+  setDriver(driver) {
+    this._driver = driver;
+
+    return this;
+  }
+
+  /**
+   * @param {Function} onLoaded
+   * @param {Function} onFail
+   */
+  load(onLoaded = () => {}, onFail = () => {}) {
+    onLoaded && this._driver.onLoadedCb(onLoaded);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/MitocGroup/deep-framework/file/2201397552/issues/source?bid=2895475&fileBranchId=3169716#l54)